Makefile

```diff
@@ -0,0 +1,57 @@
+# -- Docker
+# Get the current user ID to use for docker run and docker exec commands
+DOCKER_UID           = $(shell id -u)
+DOCKER_GID           = $(shell id -g)
+DOCKER_USER          = $(DOCKER_UID):$(DOCKER_GID)
+COMPOSE              = DOCKER_USER=$(DOCKER_USER) docker compose
+COMPOSE_RUN          = $(COMPOSE) run --rm
+
+# ==============================================================================
+# RULES
+
+default: help
+
+# -- Docker/compose
+bootstrap: ## bootstrap project
+bootstrap: \
+	build \
+	jupytext--to-md \
+	run
+.PHONY: bootstrap
+
+build: ## build custom jupyter notebook image
+	@$(COMPOSE) build notebook
+.PHONY: build
+
+down: ## stop and remove backend containers
+	@$(COMPOSE) down
+.PHONY: down
+
+jupytext--to-md: ## convert local ipynb files into md
```

Comment:
I didn't notice the first time: why the double dash?

Reply:
It was for visual purpose. As I have also the inverse command (-to-ipynb), I wanted this to be visually understandable.